pokemongo_bot/test/step_walker_test.py

```diff
@@ -91,9 +92,9 @@ def test_small_distance_small_step(self):
         self.bot.config.walk_max = 1
         self.bot.config.walk_min = 1
 
-        sw = StepWalker(self.bot, 1e-5, 1e-5)
-        self.assertEqual(sw.dLat, 0)
-        self.assertEqual(sw.dLng, 0)
+        sw = StepWalker(self.bot, 1e-5, 1e-5, precision=0.0)
+        self.assertEqual(sw.dest_lat, 1e-5)
+        self.assertEqual(sw.dest_lng, 1e-5)
```

Comment:
@th3w4y     
th3w4y added a line comment 15 minutes ago • edited
BTW:

This assert will always be true

self.assertEqual(sw.dest_lng, 1e-5)
self.lat is the position after update